lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -458,6 +458,34 @@ def test_valid_range_and_valid_min_valid_max_provided(self):
                 saver.check_attribute_compliance(self.container, self.data)
 
 
+class Test__cf_coord_identity(tests.IrisTest):
+    def check_call(self, coord_system, units, expected_units):
+        coord = iris.coords.DimCoord([30, 45], 'latitude', units=units,
+                                     coord_system=coord_system)
+        result = Saver._cf_coord_identity(coord)
+        self.assertEqual(result, (coord.standard_name, coord.long_name,
+                                  expected_units))
+
+    def test_geogcs(self):
+        crs = iris.coord_systems.GeogCS(60, 0)
+        self.check_call(coord_system=crs, units='degrees',
+                        expected_units='degrees_north')
+
+    def test_no_coord_system(self):
+        self.check_call(coord_system=None, units='degrees',
+                        expected_units='degrees_north')
+
+    def test_rotatedgeogcs(self):
+        crs = iris.coord_systems.RotatedGeogCS(30, 175)
+        self.check_call(coord_system=crs, units='degrees',
+                        expected_units='degrees')
+
+    def test_crs_with_no_default_units(self):
```

Comment:
@lbdreyer This is really a `pass-thru` rather than `default` behaviour ... what do you think?